data/dbus/Makefile

```diff
@@ -14,18 +14,14 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 BINDIR := /usr/bin
-LIBEXECDIR := /usr/lib
 DBUSDIR = /usr/share/dbus-1
 DBUSSERVICESDIR := ${DBUSDIR}/services
 
 SERVICES_GENERATED := $(patsubst %.service.in,%.service,$(wildcard *.service.in))
 SERVICES := ${SERVICES_GENERATED}
 
 %.service: %.service.in
-	cat $< | \
-		sed s:@libexecdir@:$(LIBEXECDIR):g | \
-		sed 's:@bindir@:${BINDIR}:g' | \
-		cat > $@
```

Comment:
Has this become used by anything else since it was added?

Reply:
Thanks for clearing this up